src/config.ts

```diff
@@ -246,6 +292,22 @@ export class Configuration {
             );
           }
           break;
+
+        case "create-prereleases":
+          /* Example YAML
+           *   create-prereleases: ""
+           *   create-prereleases: "dev"
+           */
+          if (typeof data[key] === "string") {
+            this.createPrereleases = data[key];
+          } else {
+            throw new Error(
+              `Incorrect type '${typeof data[
+                key
+              ]}' for '${key}', must be '${typeof this.initialDevelopment}'!`
```

Comment:
Why is this connected to the type of `initialDevelopment`?

Reply:
Oh, no reason.. It certainly wasn't a copy-paste error! :sweat_smile: